task6/task6.py

```diff
@@ -0,0 +1,45 @@
+import sys
+
+
+class SkipThisFile(Exception):
+    """Tells the generator to jump to the next file in list."""
+    pass
+
+
+def read_lines(files_):
+    for file in files_:
```

Comment:
Hy, Stas. Why you often use names with underscore? 
When you use it for input_  it is possible, but unusual. 
When you use it for files_ it is very strange.